src/elements/element.arc.js

```diff
@@ -227,10 +226,10 @@ function drawBorder(ctx, element, offset) {
   }
 
   if (inner) {
-    clipArc(ctx, element);
+    clipArc(ctx, element, element.endAngle);
```

Comment:
Is it expected to use `element.endAngle` here?

Reply:
You're right, that one is wrong. Not sure why any tests didn't get this. Maybe the clipping works properly even when the arc is over 2*PI. 
I'll update anyway.